.eslintrc

```diff
@@ -90,10 +90,10 @@
     "no-use-before-define": 2,
     "radix": 2,
     "curly": 2,
-    "no-fallthrough": 0,
-    "default-case": 0,
-    "no-plusplus": 0,
-    "dot-notation": 0
+    "no-fallthrough": 2,
+    "default-case": 2,
+    "no-plusplus": 2,
+    "dot-notation": 2
```

Comment:
We discussed this at length. You need to be able to use bracket notation to index into JSON with keys that are not valid JS identifiers like `data['4chan']`.